utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java

```diff
@@ -87,4 +90,8 @@ public static LocalPortForward portForward(Pod pod, int remotePort, int localPor
         }
     }
 
+    public static Optional<Pod> getPodByPartialName(String partialName){
+        return OpenShiftUtils.getInstance().getPods().stream().filter(p -> p.getMetadata().getGenerateName().contains(partialName)).findFirst();
```

Comment:
Please add an assert here that after your filter there is at least one instance before calling `findFirst()`